xml_inscriptions/transcribed/KY.Lou.SAM.L.1929.17.316.xml

```diff
@@ -111,7 +112,18 @@
    <text>
       <body>
          <div type="edition">
-         <!--The marked up transcription goes here, surrounded by <ab> </ab> elements-->
+      <ab>
```

Comment:
Some more markup for the names in the transcription might be needed but otherwise looks good